cmd/installer/cli/install.go

```diff
@@ -426,6 +435,7 @@ func runManagerExperienceInstall(ctx context.Context, flags InstallCmdFlags, rc
 		ManagerPort:   flags.managerPort,
 		License:       flags.licenseBytes,
 		AirgapBundle:  flags.airgapBundle,
+		AirgapInfo:    airgapInfo,
```

Comment:
Is there some reason you chose not to make airgapInfo a property of InstallCmdFlags as [suggested](https://github.com/replicatedhq/embedded-cluster/pull/2336#discussion_r2157646365)? That way you will be following the existing pattern and dont have to pass it as an additional argument to many functions.